packages/git/src/common/git-model.ts

```diff
@@ -163,6 +165,10 @@ export namespace Repository {
         }
         return repository === repository2;
     }
+    export function getRepositoryRelativePath(repository: Repository, uri: URI) {
```

Comment:
Finally, it is useful. 👍 Please add a few tests. Also, could you generalize it a bit, so it would accept strings for both arguments, and explicitly declare the return type, `string`.? One more tiny remark, since you are already in the context of `Repository`, you can rename the method to `relativePath(Repository|string, URI|string): string`. What's your take on that?